lib/rules/property-no-unknown/index.mjs

```diff
@@ -21,9 +22,16 @@ const meta = {
 	url: 'https://stylelint.io/user-guide/rules/property-no-unknown',
 };
 
+const knownProperties = [
+	...properties,
+	...legacyProperties,
+	// postcss doesn't distinguish between descriptors and properties
+	'navigation',
```

Comment:
[ask] Is it possible to add `navigation` to `known-css-properties` upstream?

Reply:
If they consider descriptors not to be properties we run the risk that it would be rejected.